lib/msal-browser/src/broker/nativeBroker/NativeRequest.ts

```diff
@@ -31,6 +31,7 @@ export type NativeTokenRequest = {
     extendedExpiryToken?: boolean;
     extraParameters?: StringDict;
     storeInCache?: StoreInCache; // Object of booleans indicating whether to store tokens in the cache or not (default is true)
+    signPopToken?: boolean;
```

Comment:
Why do we need this?

Reply:
This parameter is used to determine if MSAL JS needs to sign the pop token or not ( only sign it in cases where MSAL JS itself is creating the kid)

Reply:
I will add a comment to explain the requirement